elasticsearch/src/http/request.rs

```diff
@@ -9,10 +9,31 @@ use serde::Serialize;
 /// expect JSON, however, there are some APIs that expect newline-delimited JSON (NDJSON).
 /// The [Body] trait allows modelling different API body implementations.
 pub trait Body {
+    /// A ready-made immutable buffer that can be used to avoid writing
```

Comment:
```suggestion
    /// An existing immutable buffer that can be used to avoid having to write to another buffer
    /// that will then be written to the request stream
```

Reply:
And a `.` 🙃